tools/wpt/testfiles.py

```diff
@@ -133,7 +133,10 @@ def compile_ignore_rule(rule: Text) -> Pattern[Text]:
     return re.compile("^%s$" % "/".join(re_parts))
 
 
-def repo_files_changed(revish: Text, include_uncommitted: bool = False, include_new: bool = False) -> Set[Text]:
+def repo_files_changed(wpt_root: Text,
```

Comment:
naming nit: How about `repo_root` so that it doesn't shadow the global `wpt_root`? Conceptually, it also seems more accurate.